payroll-processor-functions/src/payroll-processor-functions/Features/Employees/EmployeeEntity.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.WindowsAzure.Storage.Table;
+using Newtonsoft.Json;
+
+namespace PayrollProcessor.Functions.Features.Employees
+{
+    /// <summary>
+    /// The table storage representation of an employee
+    /// </summary>
+    public class EmployeeEntity : TableEntity
+    {
+        public string Department { get; set; } = "";
+        public DateTimeOffset EmploymentStartedOn { get; set; }
+        public string FirstName { get; set; } = "";
+        public string LastName { get; set; } = "";
+        public string Phone { get; set; } = "";
+        public string Status { get; set; } = "";
+        public string Title { get; set; } = "";
+        public string Payrolls { get; set; } = "";
```

Comment:
`Payrolls` is a collection of `Payroll` serialized into a string.

The current business logic stores the 30 initially added `Payrolls` to this set, but I should fix that to be the 30 most recent.

Reply:
Most recent makes sense to me